tests/src/test/java/alluxio/cli/ConfigurationDocGeneratorTest.java

```diff
@@ -46,19 +47,41 @@
   @Rule
   public TemporaryFolder mFolder = new TemporaryFolder();
   @Parameterized.Parameter
+  public String mFileType;
+  @Parameterized.Parameter(1)
   public Pair<PropertyKey, String> mTestConf;
   private String mLocation;
 
+//  private Type type;
+
   @Parameters
-  public static Object[] data() {
-    return new Object[]{
-        new Pair<>(PropertyKey.USER_LOCAL_READER_PACKET_SIZE_BYTES, "user-configuration.csv"),
-        new Pair<>(PropertyKey.MASTER_CONNECTION_TIMEOUT_MS, "master-configuration.csv"),
-        new Pair<>(PropertyKey.WORKER_DATA_FOLDER, "worker-configuration.csv"),
-        new Pair<>(PropertyKey.SECURITY_AUTHENTICATION_TYPE, "security-configuration.csv"),
-        new Pair<>(PropertyKey.KEY_VALUE_PARTITION_SIZE_BYTES_MAX, "key-value-configuration.csv"),
-        new Pair<>(PropertyKey.INTEGRATION_WORKER_RESOURCE_MEM, "common-configuration.csv")
-    };
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][]{
+        {"CSV", new Pair<>(PropertyKey.USER_LOCAL_READER_PACKET_SIZE_BYTES,
```

Comment:
typically we don't use String literals for this purpose. One common practice is to define a String constant `CSV = "CSV"` as well as `YML = "YML"`, and reference the constant rather than hard coding the string literals in the code. One advantage is it is easy to refactor the code (e.g., change the name), and avoid typo when inlining the strings.